src/actions/file-search.js

```diff
@@ -94,17 +95,17 @@ export function searchContents(query: string, editor: Object) {
     const modifiers = getFileSearchModifiers(getState());
     const selectedSource = getSelectedSource(getState());
 
-    if (
-      !query ||
-      !editor ||
-      !selectedSource ||
-      !selectedSource.text ||
-      !modifiers
-    ) {
+    if (!editor || !selectedSource || !selectedSource.text || !modifiers) {
       return;
     }
 
     const ctx = { ed: editor, cm: editor.codeMirror };
+
+    if (!query) {
+      clearSearch(ctx.cm, query);
+      return;
+    }
+
```

Comment:
i wonder if we can create a unit test for this
57c385b [Tabs] Prevent :formatted from tab names, only show mappedSource footer for original sources (#6618)
70cedc6 [Editor] Footer messaging should have consistent left spacing (#6613)

Reply:
@jasonLaster Do you mean I should add a unit test method for this fix?
Sorry this is my very first PR on a projet, I need some clarification.